stable/dynamic-gateway-service/templates/statefulset.yaml

```diff
@@ -44,6 +44,9 @@ spec:
         runAsNonRoot: false
         runAsUser: 0
       affinity:
+{{- if .Values.affinity }}
+{{ toYaml .Values.affinity | indent 8 }}
```

Comment:
I'm not seeing where this `.Values.affinity` variable is being pulled.

Reply:
As I showed in the description, this would be specified using a values file at deployment time:
```
affinity:
  podAntiAffinity:
    requiredDuringSchedulingIgnoredDuringExecution:
      - topologyKey: "failure-domain.beta.kubernetes.io/zone"
        labelSelector:
          matchLabels:
            app: dynamic-gateway-service
```

Reply:
In that case, I'm good with these changes. It allows a user with the know-how to define their own affinity in the `values.yaml` without the ability being advertised.